docs/installation/reference/import.rst

```diff
@@ -0,0 +1,177 @@
+Open Zaak exposes several API endpoints to help aid the user to import larger amounts
+of `EnkelvoudigInformatieObject`'s.
+
+Process
+-------
+
+The import process consists of several steps with each step having a API endpoint.
+The process consists of the following steps:
+
+1. Creating the `Import`
+2. Uploading the `Import` metadata file
+3. (Optionally) Retrieving the status of the `Import`
+4. (Optionally) Retrieving the report file of the `Import`
+
+**Permissions**
+
+Importing `EnkelvoudigInformatieObjecten` is not possible for all authenticated users
+for Open Zaak. The reasoning behind this is that not all users for example have
+the rights to create `EnkelvoudigInformatieObjecten` with certain `InformatieObjectType`'s.
+Various permissions exist within Open Zaak and therefore to make use of the import
+functionality, users only with the `heeft_alle_autorisaties` set to `True` are allowed
+to perform any import operation.
+
+**Creating an Import**
+
+The first step in the import process is that of making an `Import` resource.
+This is done through a `POST` request. The request body of this request can be empty
+for this step.
+
+Creating an `Import` is only possible whenever no other `Import` instance exist
+with the statusses `pending` or `active`.
+
+Whenever no other import is pending or active, the endpoint will provide the user
+three urls; an url to upload an import metadata file, an url to retrieve the status
+of an import and another url for downloading a report of the import. These three
+urls provide the user the ability to progress further in the import process.
+
+The next step in the import process, after creating an `Import` instance, is providing
+a metadata file which starts the actual importing of `EnkelvoudigInformatieObject`'s.
+
+**Starting an Import**
+
+After creating an `Import` instance, users can provide an import metadafile. This
+should be a csv which consists of rows with data needed to create an `EnkelvoudigInformatieObject`.
+The data this csv should contain is rougly the same as that is needed for creating
+an `EnkelvoudigInformatieObject` through a "regular" API call with some exceptions.
+For more details about the format and the requirements of the csv file, the API
+documentation should be consulted. The request for this endpoint should be a `POST`
+request containing the csv data in its request body.
+
+Whenever the csv contains invalid and or missing headers the import process will
+not be started. Whenever headers are missing the error response will contain those
+headers.
+
+One column which is required for each row in the csv file is the path to the
+file which will be imported and will be assosiated to the `EnkelvoudigInformatieObject`.
+This should be a relative path from the path configured in `IMPORT_DOCUMENTEN_BASE_DIR`.
+
+Before starting an import, the directory configured through `IMPORT_DOCUMENTEN_BASE_DIR`,
```

Comment:
Could you add `IMPORT_DOCUMENTEN_BASE_DIR` and `IMPORT_DOCUMENTEN_BATCH_SIZE` to the environment variable configuration (`docs/installation/config/env_config.rst`) and add references for them here? 

```rst
.. _import_documenten_base_dir:
* ``IMPORT_DOCUMENTEN_BASE_DIR``: ...
```

and then refer to them in this file like this:

```rst
:ref:`IMPORT_DOCUMENTEN_BASE_DIR <import_documenten_base_dir>`
```

Not 100% sure if this example I gave works right away, but more on this can be found here https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#cross-referencing-using-roles

I also think I would maybe reference these envvars in a separate section before ``Process`` (maybe something like ``Prerequisites``) and mention that these need to be set before this functionality can be used (though I'm not sure if they need to be set explicitly or if the defaults work too), instead of first mentioning them in the process description.